ampligraph/evaluation/protocol.py

```diff
@@ -444,7 +444,7 @@ def to_idx(X, ent_to_idx, rel_to_idx):
 
 
 def evaluate_performance(X, model, filter_triples=None, verbose=False, filter_unseen=True, entities_subset=None,
-                         corrupt_side='s,o', use_default_protocol=False):
+                         corrupt_side='s,o', ranking_strategy='worst', use_default_protocol=False):
```

Comment:
A side comment, but I thought we had removed the use_default_protocol parameter at some point?

Reply:
We haven't yet.